src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -781,6 +788,9 @@ protected DynamoModel(IStartConfiguration config)
                     var isFirstRun = PreferenceSettings.IsFirstRun;
                     PreferenceSettings = migrator.PreferenceSettings;
 
+                    //Set the DynamoModel.PreferenceSetting as a static reference to the PreferenceSetting class for use in PreferenceSettings.Instance
+                    PreferenceSettings.dynamoModelRuntimePreferenceSettings = PreferenceSettings;
```

Comment:
Just curious why do we need to set this again here if the reference is already pass once above?

Reply:
You are correct.  Let me remove that one